Source/Core/Core/WiiRoot.cpp

```diff
@@ -50,8 +53,57 @@ void ShutdownWiiRoot()
 {
   if (!s_temp_wii_root.empty())
   {
+    std::string save_path =
+        Common::GetTitleDataPath(Movie::GetTitleId(), Common::FROM_SESSION_ROOT);
+    std::string user_save_path =
+        Common::GetTitleDataPath(Movie::GetTitleId(), Common::FROM_CONFIGURED_ROOT);
+    std::string user_backup_path =
```

Comment:
Would it be make sense to have a function for getting a title data path from the backup directory? I'm just asking -- feel free to ignore this